specifyweb/frontend/js_src/lib/components/WbToolkit/WbToolkit.tsx

```diff
@@ -101,7 +99,7 @@ export function WbToolkit({
       )}
       <WbLeafletMap
         hasLocality={hasLocality}
-        hot={hot}
+        hot={hot!}
```

Comment:
this doesn't seem right
since `WbLeafletMap` shouldn't be called with `hot` being undefined, you should not be lying to typescript about `hot` being always defined here.
instead, conditionally render WbLeafletMap only when `hot` is defined: `{typeof hot === 'object' && <WbLeafletMap ... />}`

Reply:
What do you think of conditionally rendering `WbToolkit` entirely when `hot` is defined instead of conditionally rendering the individual components inside it?

Reply:
sounds good!